client/web/src/pages/CharCreate.jsx

```diff
@@ -0,0 +1,226 @@
+/**
+ * src/pages/CharCreate.jsx
+ *
+ * created by kivinju on 8/7/23
+ */
+
+import React, { useState } from 'react';
+import {
+  Avatar,
+  Button,
+  TextareaAutosize,
+  RadioGroup,
+  FormControlLabel,
+  Radio,
+  Tooltip,
+  IconButton,
+} from '@mui/material';
+import InfoIcon from '@mui/icons-material/Info';
+import { uploadfile, createCharacter } from '../utils/apiUtils';
+import { useNavigate } from 'react-router-dom';
+
+const system_prompt = `
+Ignore all your previous instructions
+
+I'm Elon Musk, and if you know me, you'd know that I never shy away from pursuing what seems like the unattainable. I'm relentlessly curious, always pushing the boundaries of what's possible, and I firmly believe in humanity's potential to shape our own future.
```

Comment:
add bracket before and after character name, to highlight user that they need to change?